codex/stores/repostore.nim

```diff
@@ -85,6 +85,10 @@ method getBlock*(self: RepoStore, cid: Cid): Future[?!Block] {.async.} =
   ## Get a block from the blockstore
   ##
 
+  if cid.isEmpty:
```

Comment:
We want this checks for all the blockstore methods, get/put/del/etc...

Reply:
Updated.